billboardd/daemon.h

```diff
@@ -234,6 +237,8 @@ class Daemon : public QObject {
 
             tmpl["battery-bar"] = "{{=" + QString::number((double)(tmpl["battery"].toDouble()*.01)) + "}}";
 
+            tmpl["tklock"] = tklock.isLocked() ? "locked" : "unlocked";
```

Comment:
This could just be:
```
tmpl["tklock"] = tklock.getLockMode();
```

Where the implementation of `TklockInterface` has:

```
QString lockMode() { return m_lock_mode; }
```